Calendar/BookingHelpers/BookingMessageTrackingLogLogic.ps1

```diff
@@ -0,0 +1,27 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function RunMessageTrackingLogValidation {
+    [ref]$ErrorMessage = $null
+    [ref]$WriteMessageAlways = $false
+    $TestResult = $true
+
+    Write-DashLineBoxColor "Running Message Tracking Log Validation" -Color Blue
+
+    $ErrorMessage = ""
+    $TestResult = CheckMessageTrackingLogs -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+    WriteTestResult "Collecting Message tracing logs " -success $TestResult -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+}
+
+function CheckMessageTrackingLogs {
+    param([ref]$ErrorMessage)
+    Write-Verbose "Collect Message tracking logs for Booking MB"
+    if ($null -eq $script:MessageTrackingLogs) {
+        $ErrorMessage.Value ="Message Tracking Logs are null "
+        return $false
+    }
+
+    $ErrorMessage.Value ="Message Tracking Logs contains " + $script:MessageTrackingLogs.Count + " entries"
+    $WriteMessageAlways.Value = $true
+    return $true
+}
```

Comment:
Is there a reason why this isn't in the parent function?

Reply:
This is just one of the future tests on Tracking Logs. All tests have a separate method. This one is only checking if there are MT logs or not, future tests will involve running more logic on MT Logs.